package.json

```diff
@@ -16,6 +16,10 @@
     ".": {
       "types": "./dist/index.d.ts",
       "default": "./dist/index.js"
+    },
+    "./internal": {
+      "types": "./dist/internal.d.ts",
+      "default": "./dist/internal.js"
```

Comment:
Want to clearly delineate public and internal APIs, so `@eppo/js-client-sdk-common/internal` is intended for internal SDK usage only and should not be re-exposed to users.